src/main/webapp/WEB-INF/tags/student/feedbackResults/questionWithResponses.tag

```diff
@@ -13,10 +13,20 @@
 
     <c:if test="${questionWithResponses.questionDetails.individualResponsesShownToStudents}">
 
-      <c:forEach items="${questionWithResponses.responseTables}" var="responseTable">
-        <feedbackResults:responseTable responseTable="${responseTable}"/>
+      <c:forEach items="${questionWithResponses.othersResponseTables}" var="othersResponseTable">
+        <feedbackResults:responseTable responseTable="${othersResponseTable}"/>
       </c:forEach>
 
+      <c:if test="${!questionWithResponses.selfResponseTablesEmpty}">
```

Comment:
Personally, I prefer `isSelfResponseTablesEmpty`. It's more like a boolean variable. You can achieve this by setting getter `getIsSelfResponseTablesEmpty`